src/map/buyingstore.cpp

```diff
@@ -78,13 +78,13 @@ int8 buyingstore_setup(struct map_session_data* sd, unsigned char slots){
 		return 2;
 	}
 
-	if( map_getmapflag(sd->bl.m, MF_NOVENDING) )
+	if( map_getmapflag(sd->bl.m, MF_NOBUYINGSTORE) )
 	{// custom: no vending maps
```

Comment:
GitHub won't let me add a suggestion, but maybe rename this to "no buyingstore maps"